docs/api-testing/integrations/apitesting-saucectl-integration.md

```diff
@@ -0,0 +1,105 @@
+---
+id: apitesting-saucectl-integration
+title: CI/CD Platform Integration with saucectl
+sidebar_label: CI/CD Integration (saucectl)
+description: 'Using Sauce Labs API Testing or CLI, you can seamlessly integrate continuous API testing into your CI/CD pipeline.'
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Using the [`saucectl` CLI](/dev/cli/saucectl), you can execute API tests and interact with Sauce Labs API Testing.
+
+## What You'll Need
+
+- A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
+- Your Sauce Labs [Username and Access Key](https://app.saucelabs.com/user-settings).
+- An existing API Testing Project. For details on how to create one, see [API Testing Quickstart](/api-testing/quickstart/).
+
```

Comment:
- Have [Git](https://git-scm.com/downloads) and [GitHub](https://docs.github.com/) set up.

Reply:
That's something we never mentioned on other projets. Should we add this ?
Knowing that a Github account is not even needed, and we expect a user to know how to use git.

Reply:
thought it would  be needed to work with the example repo.